R/derive_param_maxsev_draft.R

```diff
@@ -0,0 +1,124 @@
+#' Title: DERIVE_PARAM_MAXSEV 
+#' ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ 
+#' @description                                                                 +
+#' This derive_param_maxsev function is used to derive the maximum severity for +
+#'  each administrative and Systemic events.                                    + 
+#' ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

```

Comment:
```suggestion
#' ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ 
#' @description                                                                 +
#' This `derive_param_maxsev` function is used to derive the maximum severity for +
#'  each administrative and Systemic events.                                    + 
#' ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

```

Reply:
Hi Ben, I have updated the code. Please check the latest version.